### PR TITLE
Feat/storybook community portal tags

### PR DESCRIPTION
**Pull request by osmaczko**
### What does the PR do

iterates: https://github.com/status-im/status-desktop/issues/7243

### Affected areas

Community Portal

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/33099791/196169253-d3cb34e5-360d-45f0-8440-16d54dd9dca2.mp4

### CODE REVIEW

**Review by caybro — COMMENTED** [review]
We already have those icons, as "next.svg" and "previous.svg" (although not in this color)

File: ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.14
+
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
```
Comment (caybro) [inline ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml:5]: Please use QQC2.Control for new components/controls, not Rectangle

Comment (caybro) [reply ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml:5]: Or AbstractButton even (we have a clicked() signal below, some background, image + mouse area as the `contentItem; etc. All typical Button stuff

**Review by caybro — CHANGES_REQUESTED** [review]

**Review by MishkaRogachev — CHANGES_REQUESTED** [review]

File: ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml (hunk 1 above)
Comment (osmaczko) [reply ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml:5]: I agree it should be based on QQC2, but I just moved already existing component to StatusQ: 
image

File: storybook/pages/CommunitiesPortalLayoutPage.qml [hunk 2]
```diff
@@ -25,7 +25,11 @@ SplitView {
                 readonly property string locale: ""
                 readonly property int unreadNotificationsCount: 42
                 readonly property string communityTags:
-                    JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼"})
+                    JSON.stringify({"Activism":"✊","Art":"🎨","Blockchain":"🔗","Books & blogs":"📚","Career":"💼","Collaboration":"🤝","Commerce":"🛒","Culture":"🎎","DAO":"🚀","DIY":"🔨","DeFi":"📈",
```
[the remark this answers is not in the document]
Comment (osmaczko) [reply storybook/pages/CommunitiesPortalLayoutPage.qml:28]: This is how tags were mocked for the storybook, I just extended them.

[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply storybook/pages/CommunitiesPortalLayoutPage.qml:28]: Yep, in storybook they are just copy-paster. Originally they are stored in status-go

File: ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml (hunk 1 above)
Comment (osmaczko) [reply ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml]: @caybro anyway addressed: https://github.com/status-im/status-desktop/pull/7929/commits/a8904af1f001942ffe29d2fb2de66487e88b9430

File: ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml [hunk 3]
```diff
@@ -1,46 +1,45 @@
...
-        id: d
+        // otherwise there is no pointing hand cursor when button is hovered
+        MouseArea {
+            anchors.fill: parent
+            cursorShape: Qt.PointingHandCursor
+            propagateComposedEvents: true
 
-        function iconSrc(icon) {
-            return "../../assets/img/icons/" + icon + ".svg";
+            onClicked: root.clicked()
```
Comment (caybro) [inline ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml:42]: (almost) perfect, thanks for this!

Just don't handle the clicks or composed events here, set:
```qml
acceptedButtons: Qt.NoButton
```

Comment (osmaczko) [reply ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml]: Oh, didn't know this trick, nice :ok_hand:

**Review by MishkaRogachev — APPROVED** [review]

File: storybook/pages/CommunitiesPortalLayoutPage.qml (hunk 2 above)
Comment (noeliaSD) [inline storybook/pages/CommunitiesPortalLayoutPage.qml:28]: @osmaczko does it make sense to have it in a store or maybe directly in the backend instead?

**Pull request merged**